microcosm_flask/tests/conventions/test_health.py

```diff
@@ -123,3 +123,76 @@ def fail(graph):
             },
         },
     })))
+
+
+def test_health_check_optional_custom_check():
```

Comment:
These seem like they could be parameterized. Also a test explicitly setting full to False would be a nice to have

Reply:
Sure.